### ISSUE TITLE
Add support for TEALv6 opcodes

### ISSUE THREAD

**Issue by geo2a** [crytic/tealer#78]
Is there a plan to work on adding support for the new TEALv6 opcodes?

I'm primarily interested to be able to generate control-flow graphs of TEALv6 programs with Tealer, hence I suggest adding:

* Representation of TEALv6 instructions into [`tealer/teal/instructions/instructions.py`](https://github.com/crytic/tealer/blob/dev/tealer/teal/instructions/instructions.py)
* Representation of Account Parameters Fields into a new file,  `tealer/teal/instructions/instructions.py`, which are required for the `acct_params_get f` [instruction](https://developer.algorand.org/docs/get-details/dapps/avm/teal/opcodes/?from_query=acct#acct_params_get-f).
* New parsing rules into [`tealer/teal/instructions/parse_instruction.py`](https://github.com/crytic/tealer/blob/dev/tealer/teal/instructions/parse_instruction.py)
* Relevant tests

I'm eager to work on this, but I wouldn't want to scoop other's effort, if somebody already planning to work on it.

**Comment by dguido** [comment]
Yes! Please, we'll even gladly compensate you for it. Email me dan@trailofbits.com when you want to start working on it (or after you're done). Keeping up with new language features is always a challenge and we appreciate the help.
